src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,348 @@
+   */
+  @property({
+    attribute: 'person-details',
+    type: Object
+  })
+  public personDetails: MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact;
+  /**
+   * Set the image of the person
+   * Set to '@' to look up image from the graph
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'person-image',
+    type: String
+  })
+  public personImage: string;
+
+  /**
+   * Gets or sets whether additional details section is rendered
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'is-extended',
+    type: Boolean
+  })
+  public isExtended: boolean = false;
+
+  /**
+   * Synchronizes property values when attributes change.
+   *
+   * @param {*} name
+   * @param {*} oldValue
+   * @param {*} newValue
+   * @memberof MgtPersonCard
+   */
+  public attributeChangedCallback(name, oldValue, newValue) {
+    super.attributeChangedCallback(name, oldValue, newValue);
+
+    if (name === 'is-extended' && oldValue !== newValue) {
+      this.isExtended = false;
+    }
+  }
+
+  /**
+   * Invoked when the element is first updated. Implement to perform one time
+   * work on the element after update.
+   *
+   * Setting properties inside this method will trigger the element to update
+   * again after this update cycle completes.
+   *
+   * * @param _changedProperties Map of changed properties with old values
+   */
+  public firstUpdated() {
+    Providers.onProviderUpdated(() => this.loadData());
+    this.loadData();
+  }
+
+  /**
+   * Invoked on each update to perform rendering tasks. This method must return
+   * a lit-html TemplateResult. Setting properties inside this method will *not*
+   * trigger the element to update.
+   */
+  protected render() {
+    if (this.personDetails) {
+      const user = this.personDetails;
+
+      // tslint:disable-next-line: one-variable-per-declaration
+      let department, jobTitle;
+
+      if (user.department) {
+        department = html`
+          <div class="department">${user.department}</div>
+        `;
+      }
+
+      if (user.jobTitle) {
+        jobTitle = html`
+          <div class="job-title">${user.jobTitle}</div>
+        `;
+      }
+      return html`
+        <div class="root" @click=${this.handleClose}>
+          <div class="default-view">
+            ${this.renderTemplate('default', { person: this.personDetails }) ||
+              html`
+                <mgt-person .personDetails=${this.personDetails} .personImage=${this.personImage}></mgt-person>
+                <div class="details">
+                  <div class="display-name">${user.displayName}</div>
+                  ${jobTitle} ${department}
+                  <div class="base-icons">
+                    ${this.renderIcons()}
+                  </div>
+                </div>
+              `}
+          </div>
+          <div class="additional-details-container" @click=${this._showAdditionalDetails}>
```

Comment:
shouldn't this click handler only be available if not expanded? Consider moving the click handler in the _showAdditionalDetails function

Reply:
moved onclick to renderAdditionalDetails, where the button itself lives section: e6978a7